lib/widgets/message_list.dart

```diff
@@ -629,6 +623,24 @@ class _UnreadMarker extends StatelessWidget {
   }
 }
 
+class _AnimatedUnread extends StatelessWidget {
+  const _AnimatedUnread({required this.isRead, required this.child});
```

Comment:
```suggestion
  const _AnimatedVisibilityEaseOut({required this.visible, required this.child});
```

maybe a better name would be `_AnimatedVisibilityEaseOut`, since it takes a `Widget child` and that seems much more general interface than anything related to "unread".